### PR TITLE
Update SwiftLint config file in 7-code-style.md

### PR DESCRIPTION
**Pull request by AlexanderKulyk**
- Updates SwiftLint recommended config file link in Code Style chapter.
- Adds **.swiftlint.yml** file to repo. It includes latest rules from 0.49.0.

### CODE REVIEW

**Comment by AlexanderKulyk** [comment]
Hey @ANDaleksei, @ArtemMysik, and @SaladDays831.
I tagged you to hear your thoughts and feelings about the specific ruleset I composed.
These rules will be used to set up upcoming projects.
If you have any questions or ideas, or you don’t want to include a specific rule - please LMK.

File: resources/.swiftlint.yml [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+force_try: warning
+
+force_unwrapping: error
+
+function_body_length:
+  warning: 100
+  error: 120
+
+identifier_name:
+  excluded:
```
Comment (ANDaleksei) [inline resources/.swiftlint.yml:139]: I have several more short arguments: `by`, `to`.

File: resources/.swiftlint.yml [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+force_try: warning
+
+force_unwrapping: error
+
+function_body_length:
+  warning: 100
+  error: 120
+
+identifier_name:
```
Comment (ANDaleksei) [inline resources/.swiftlint.yml:138]: I also have similar `type_name` with excluded `ID` which I use for phantom types: `public typealias ID = Tagged<User, String>` for example.

**Review by ANDaleksei — APPROVED** [review]
Nice work with SwiftLint rules, I'll copy-paste it to my project)

File: resources/.swiftlint.yml (hunk 1 above)
Comment (AlexanderKulyk) [reply resources/.swiftlint.yml:139]: Adding this values will allow their initialization like this:
```
let by = valueOne - valueTwo
let to = getTargetObject(for: data)
```
And they look rather confusing.
Can you please bring some examples how you use them.

Comment (AlexanderKulyk) [reply resources/.swiftlint.yml:139]: `identifier_name` rule is not triggered by argument labels:

Screenshot 2022-08-22 at 18 24 30

If you use them as such - I thinks you can try removing these exemptions from your local config.

Comment (ANDaleksei) [reply resources/.swiftlint.yml:139]: Yes, you're right.

File: resources/.swiftlint.yml [hunk 3]
```diff
@@ -0,0 +1,160 @@
+# Revised for 0.48.0
```
Comment (ArtemMysik) [inline resources/.swiftlint.yml:1]: I would also add https://realm.github.io/SwiftLint/rule-directory.html as a comment for reference

**Review by ArtemMysik — APPROVED** [review]
LGTM!

**Review by SaladDays831 — APPROVED** [review]
🔥

**Pull request merged**